### ISSUE TITLE
Tailwindcss

### ISSUE THREAD

**Issue by alexfalconflores** [zed-industries/extensions#453]
### Check for existing issues

- [x] Completed

### Language

Tailwindcss

### Misc notes

I would like you to add a plugin for tailwindcss, that behaves like the extension that is available in VSCode.

**Comment by Moshyfawn** [comment]
Zed core has the TW support built-in.

**Comment by alexfalconflores** [comment]
In VSCode when you hover over the tailwind class a tooltip appears showing how to compile to css, but not in zed.
_vscode_
![image](https://github.com/zed-industries/extensions/assets/75406302/8dd9454f-a73c-4080-8cf5-5677c710dbc5)

**Comment by mariansimecek** [comment]
@alexfalconflores 
It was fixed in [0.130.4](https://zed.dev/releases/stable/0.130.4)
https://github.com/zed-industries/zed/issues/7947
![image](https://github.com/zed-industries/extensions/assets/5357955/7be7f79a-7e2a-4f10-8088-0a53f23a129e)

**Comment by alexfalconflores** [comment]
oh, great, thanks ❤️

**Comment by alexfalconflores** [comment]
@mariansimecek 
This is also missing to complete the use of Tailwind.
![image](https://github.com/zed-industries/extensions/assets/75406302/1e2f98d9-5a02-4e8b-9fc2-c7935d7b69a5)

**Comment by henrikruscon** [comment]
Bit more elegant solution:
![CleanShot 2024-04-14 at 22 00 26@2x](https://github.com/zed-industries/extensions/assets/1430576/c64707a9-15a1-4455-9269-230f8d4ddd68)

Definitely a feature I rely upon all the time.

**Comment by alexfalconflores** [comment]
what editor do you use? @henrikruscon

**Comment by henrikruscon** [comment]
> what editor do you use? @henrikruscon

Panic Nova

**Comment by alexfalconflores** [comment]
Yes it looks good, but a dialog next to it could contain more information and documentation.

**Comment by alexfalconflores** [comment]
> > what editor do you use? @henrikruscon
> 
> Panic Nova

Yes it looks good, but a dialog next to it could contain more information and documentation.
